vnext/Microsoft.ReactNative.Managed/Microsoft.ReactNative.Managed.csproj

```diff
@@ -101,8 +101,7 @@
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="..\Microsoft.ReactNative.SharedManaged\*.cs" />
+    <Compile Include="**\*.cs" />
```

Comment:
I've seen VS choke when doing wildcards instead of listing all the files... are we sure this is cool? :)